docs/optimization_guide/nncf/ptq/code/ptq_torch_fx.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#! [dataset]
+import nncf
+import torch
+
+calibration_loader = torch.utils.data.DataLoader(...)
+
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+calibration_dataset = nncf.Dataset(calibration_loader, transform_fn)
+#! [dataset]
+
+#! [quantization]
+import torchvision
+import openvino.torch
+model = torchvision.models.resnet50(pretrained=True)
+
+input_fp32 = ... # FP32 model input
```

Comment:
I suggest creating a sample input data for exporting the model.

Reply:
Should I update this line for every example? 
![image](https://github.com/user-attachments/assets/ceaf92b8-6069-4ca7-8575-a4fbee979f3e)
![image](https://github.com/user-attachments/assets/a826d8cb-c23f-4935-a5eb-17a3473431b3)